src/imagery/i.eodag/i.eodag.py

```diff
@@ -41,21 +43,21 @@
 
 # %flag
 # % key: e
-# % description: Extract the downloaded the datasets, not considered unless provider is set
+# % description: Extract the downloaded the scenes, not considered unless provider is set
 # %end
 
 # %flag
 # % key: d
 # % description: Delete the product archive after downloading, not considered unless provider is set
```

Comment:
What product will be deleted? and why is it dependent on setting a provider? i.eodag only downloads products, it won't be importing them, so why would the user use this flag to delete the downloaded files? Such flag makes more sense when a tool unzips and imports data into grass, but IMO, that's out of scope for i.eodag

Reply:
Setting `delete_archive` to `Yes` will delete the ZIP files  if they are extracted. (ZIP files aka. archive files in the eodag context)

So for example you could say `extract=Yes` `delete_archive=No` that will unzip the zip files without deleting them.
Another case would be let's say, `extract=Yes` `delete_archive=Yes` here it will unzip the files and delete the archive.

I added these `extract` and `delete_archive` options because it is an integral part of the eodag config file, so if the user messes something up in the `edoag.yml`, we could use these options to guarantee the intended behavior.